### DISCUSSION TITLE
I need help with transparent cubes.

### DISCUSSION THREAD

**Discussion by Kerbal-Man** [raysan5/raylib#2724] [Q&A]
I want to draw some cubes with an option of how opaque it will be. Does anyone know how to do this?

**Comment by raysan5** [comment]
@Kerbal-Man When you define the tint color, you can change the alpha with `Fade()` function, alpha goes from 0.0f (completely transparent) to 1.0f (completely opaque). 

Here an example to draw a half-transparent RED cube.
```c
DrawCube(cubePosition, 2.0f, 2.0f, 2.0f, Fade(RED, 0.5f));
```

**Comment by chriscamacho** [comment]
Do take care, you need to depth sort the tris of your cube or you'll get some odd results, you'll also need to draw everything behind the cubes too...
